alphastats/DataSet_Plot.py

```diff
@@ -364,48 +161,51 @@ def plot_dendrogram(
         )
 
         fig = plotly_object(fig)
-        fig = self._update_figure_attributes(
-            figure_object=fig, plotting_data=self.mat, method="dendrogram"
+        self._update_figure_attributes(
+            fig,
+            plotting_data=self.mat,
+            preprocessing_info=self.preprocessing_info,
+            method="dendrogram",
         )
         return fig
 
-    def plot_imputed_values(self):  # not used
-        # get coordinates of missing values
-        df = self.mat
-        s = df.stack(dropna=False)
-        missing_values_coordinates = [list(x) for x in s.index[s.isna()]]
-
-        # get all coordinates
-        coordinates = list(
-            itertools.product(list(self.mat.index), list(self.mat.columns))
-        )
-
-        # needs to be speed up
-        imputed_values, original_values = [], []
-        for coordinate in coordinates:
-            coordinate = list(coordinate)
-            if coordinate in missing_values_coordinates:
-                value = self.mat.loc[coordinate[0], coordinate[1]]
-                imputed_values.append(value)
-            else:
-                original_values.append(value)
-
-        label = ["imputed values"] * len(imputed_values) + ["non imputed values"] * len(
-            original_values
-        )
-        values = imputed_values + original_values
-
-        plot_df = pd.DataFrame(
-            list(zip(label, values)), columns=["Imputation", "values"]
-        )
-
-        fig = px.histogram(
-            plot_df,
-            x="values",
-            color="Imputation",
-            opacity=0.8,
-            hover_data=plot_df.columns,
-            template="simple_white+alphastats_colors",
-        )
-
-        pass
+    # def plot_imputed_values(self):  # TODO not used
```

Comment:
This is a requested feature and should be an issue to add the interface.

Reply:
https://github.com/orgs/MannLabs/projects/12/views/1?filterQuery=multi&pane=issue&itemId=82600738